packages/edit-site/src/components/layout/index.js

```diff
@@ -228,7 +212,7 @@ export default function Layout() {
 					/>
 
 					<AnimatePresence initial={ false }>
-						{ isEditorPage && isEditing && (
+						{ canvasMode === 'edit' && (
```

Comment:
Same as above: didn't find any page with `canvasMode=edit` that wasn't the editor.

Reply:
yes, that's the point here. The header is only shown when the canvasMode is edit, I guess your remark is a bit unclear to me.

Reply:
This wasn't meant as something you have to address but as something I've checked, and I see as correct in the PR. Sorry for the confusion :)

Reply:
I'm so dumb :P